nemo/collections/tts/torch/data.py

```diff
@@ -393,6 +394,8 @@ def __getitem__(self, index):
         # Let's keep audio name and all internal directories in rel_audio_path_as_text_id to avoid any collisions
         rel_audio_path = Path(sample["audio_filepath"]).relative_to(self.base_data_dir).with_suffix("")
         rel_audio_path_as_text_id = str(rel_audio_path).replace("/", "_")
+        if sample["phoneme"] == 1:
```

Comment:
One question about the value of `sample["phoneme"]`, is it always `None` given the statement in Line 183? IIUC, the key "phoneme" is not written into .json files in your `get_data.py`, so `"phoneme"` always does not exist in `item`. Maybe we should add a line `{"phoneme": 1}` after Line 88 in your `get_data.py`? Please correct me if I am wrong.

Reply:
I add is_phoneme flag only when augmenting dataset with phonemes. See section `3. Augmenting dataset with phonemes` in `tutorials/tts/Fastpitch_Training_GermanTTS.ipynb`.